src/creatures/players/player.cpp

```diff
@@ -6701,6 +6701,14 @@ void Player::triggerTranscendance() {
 		sendBasicData();
 		wheel()->sendGiftOfLifeCooldown();
 		g_game().reloadCreature(getPlayer());
+		updateStatsAfterTranscendance = true;
+	}
+
+	if (updateStatsAfterTranscendance && wheel()->getOnThinkTimer(WheelOnThink_t::AVATAR_FORGE) <= OTSYS_TIME()) {
+		updateStatsAfterTranscendance = false;
+		sendSkills();
```

Comment:
Isn't it just moving these calls to the end of the function?
https://github.com/opentibiabr/canary/blob/f359dca3ddd300fed86e93457f6d66a6c222610b/src/creatures/players/player.cpp#L6699

This looks like code duplication to me